src/Frontend/Asset/JsCompiler.php

```diff
@@ -89,30 +81,13 @@ class_exists(Configurator::class);
 
         $minifier = new FirstAvailable;
 
-        $remoteCache = $minifier->add('RemoteCache');
-        $remoteCache->url = 'http://s9e-textformatter.rhcloud.com/flarum-minifier/';
-
-        $hostedMinifer = $minifier->add('HostedMinifier');
-        $hostedMinifer->url = 'http://s9e-textformatter.rhcloud.com/flarum-minifier/';
-        $hostedMinifer->httpClient->timeout = 30;
-
         $ccs = $minifier->add('ClosureCompilerService');
         $ccs->compilationLevel = 'SIMPLE_OPTIMIZATIONS';
         $ccs->httpClient->timeout = 30;
 
         $minifier->add('MatthiasMullieMinify');
+        $minifier->add('Noop');
```

Comment:
We are still using `MatthiasMullieMinify` here, but the dependency was removed from composer.json. In text-formatter itself, it is only a dev dependency. Mistake or what am I missing?

Reply:
Oh god, thanks for noticing it.

Reply:
I am confused. It's gone now, because of Toby's Webpack changes, right?

Reply:
It's gone from the JsCompiler, but we need to add minification into the Formatter class because nothing else is gonna minify the formatter JS.

Reply:
Doesn't it re-minify everything?

Reply:
No, JsCompiler no longer minifies anything, we've moved that responsibility to webpack so the `js/dist/*` files come pre-minified.

Reply:
I mean doesn't it re-minifiy webpack-minified version once again?
https://github.com/flarum/core/blob/3f683dd6ee739f850765e9827f8b410f9c9187f9/src/Frontend/Asset/RevisionCompiler.php#L152

Reply:
should we only minify [added strings](https://github.com/flarum/core/blob/3f683dd6ee739f850765e9827f8b410f9c9187f9/src/Frontend/Asset/RevisionCompiler.php#L64)?

Reply:
No, it doesn't: https://github.com/flarum/core/blob/3f683dd6ee739f850765e9827f8b410f9c9187f9/src/Frontend/Asset/JsCompiler.php#L39-L42

It's no longer the responsibility of the compiler to minify, it's the responsibility of the caller. Please add minification to the Formatter JS output as per http://s9etextformatter.readthedocs.io/JavaScript/Minifiers/#matthiasmullie92minify